win/rl/winrl.cc

```diff
@@ -290,6 +290,13 @@ NetHackRL::fill_obs(nle_obs *obs)
         obs->misc[0] = in_yn_function;
         obs->misc[1] = in_getlin;
         obs->misc[2] = xwaitingforspace;
+        obs->misc[3] = false;
```

Comment:
Be careful with C++ booleans like `true/false` vs C `bools`. They can in some situations not be the same thing and map to objects that evaluate to different logical values (!)

In this case it's ok because the lhs is an int, so the type will just get casted. Given that, it might make sense to write an int here in the first place.